include/swift/Sema/TypeCheckRequest.h

```diff
@@ -114,7 +114,42 @@ class TypeCheckRequest {
   }
 
 #include "swift/Sema/TypeCheckRequestPayloads.def"
-  
+
+  TypeCheckRequest(const TypeCheckRequest &T) { *this = T; }
+
+  TypeCheckRequest& operator=(const TypeCheckRequest &T) {
```

Comment:
Would this be equivalent to this? (I'm not entirely sure if that's legal, I haven't written C++ in a while)
```cpp
TypeCheckRequest(const TypeCheckRequest &T) = default;
TypeCheckRequest& operator=(const TypeCheckRequest &T) = default;
```